docker-compose/cassandra-3.11/docker-compose-dev-mode.yml

```diff
@@ -8,7 +8,6 @@ services:
     ports:
       - "9042:9042"
       - "8081:8081"
-      - "8084:8084"
```

Comment:
GH Action fails saying the port is already in use. Must be something running in that environment but was not able to determine what. So I just removed this port mapping temporarily to make progress.